src/NuGet.Core/NuGet.Commands/PushRunner.cs

```diff
@@ -50,6 +50,7 @@ public static class PushRunner
                 timeoutSeconds,
                 disableBuffering,
                 endpoint => CommandRunnerUtility.GetApiKey(settings, endpoint, apiKey),
+                symbolsEndpoint => CommandRunnerUtility.GetApiKey(settings, symbolsEndpoint, symbolsApiKey),
```

Comment:
It used to get the same nuget.org key for symbols by default, now it looks for a key associated with symbols endpoint. Doesn't it break existing customers?